tests/e2e-agents/e2e_agents/cli.py

```
@@ -0,0 +1,164 @@
+@cli.command()
+@click.option("--area", "-a", multiple=True, help="Agent area(s) to run (e.g. incidents, settings)")
+@click.option("--all", "run_all", is_flag=True, help="Run all agents")
+@click.option("--headless/--no-headless", default=True, help="Run browser headlessly")
+@click.option("--model", default=None, help="Override LLM model")
+@click.option("--max-steps", type=int, default=None, help="Override max steps per agent")
+@click.option("--base-url", default=None, help="Override target URL")
+@click.option("--skip-health-check", is_flag=True, help="Skip waiting for app to be ready")
+@click.option("--diff-context/--no-diff-context", default=True, help="Inject git diff into prompts")
+@click.option("--parallel", type=int, default=1, help="Number of agents to run in parallel")
+@click.option("--description", default=None, help="Natural language test instructions (simulates PR description)")
+def run(area, run_all, headless, model, max_steps, base_url, skip_health_check, diff_context, parallel, description):
```

Ruff (0.15.12)

[warning] Too many branches (18 > 12)

(PLR0912)